package.json

```diff
@@ -9,13 +9,16 @@
     "start": "next start"
   },
   "dependencies": {
+    "body-scroll-lock": "^4.0.0-beta.0",
```

Comment:
Do we need a whole package for this? Can't we just get current `scrollY`, disable overflow, then reverse when needed?

Reply:
(Genuine question, could be flaky)

Reply:
I had originally put an `overflow: none` on the `body` that would toggle (which this library also does) but from the looks of it it is not enough to handle all cross-platform cases. I think it is appropriate to add this library.